### DISCUSSION TITLE
[BUG] Remove-PnPContentType - Try Catch Doesn't Work Properly

### DISCUSSION THREAD

**Discussion by robertcaretta** [pnp/powershell#1255] [Q&A]
### Reporting an Issue or Missing Feature
When using Remove-PnPContentType and Remove-PnPContentTypeFromList the errors do not cause the catch action to fire properly. So I am unable to log the results properly.  If there is a work-around please let me know.

### Expected behavior 
When the cmdlets error, the catch action should trigger so I can log the action properly and/or take a different action.

### Actual behavior 
When the action fails it doesn't register as an error that can be caught by try catch.  
I am marking the site content type as editable properly before removal attempts as well. 

### Steps to reproduce behavior
![2021-10-14_10-31-41](https://user-images.githubusercontent.com/1613085/137351367-277b4811-81c3-46e2-b21d-147b9307e3d4.png)

`try {
         Remove-PnPContentType -Identity $_.Name -Force -Verbose
         $msg = "SUCCESS | Site CT Removed | $SiteURL | CT: $($_.Name)"
          Write-Host $msg
          Write-Log -LogFile $logFile -LogMessage $msg
  }
  catch {
         $msg = "ERROR | Site CT Not Removed | $SiteURL | CT: $($_.Name)"
          Write-Host $msg
          Write-Log -LogFile $logFile -LogMessage $msg
 }
finally {
         Clear-Variable msg
 }`

### What is the version of the Cmdlet module you are running?
1.8

### Which operating system/environment are you running PnP PowerShell on?
- [x] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

**Comment by veronicageek** [comment]
@robertcaretta - This is unlikely a PnP issue. 

The `Remove-PnPContentType` is not a terminating error. What is the value of your `$ErrorActionPreference` in the session? 
Try to put a `-ErrorAction Stop` at the end of the `Remove-PnPContentType` command.
